core/tests/protractor_utils/workflow.js

```diff
@@ -139,6 +139,18 @@ var createAndPublishExploration = function(
   publishExploration();
 };
 
+var createAddAndPublish = function(
```

Comment:
Pick a more descriptive name. Create, add (what?) and publish what? I think it should be something like createAddIneractionAndPublishExp()

Reply:
Done

Reply:
Done